app/uk/gov/hmrc/apiplatform/modules/approvals/services/ResponsibleIndividualVerificationService.scala

```diff
@@ -27,30 +29,26 @@ import uk.gov.hmrc.thirdpartyapplication.domain.models.{Standard, TermsOfUseAcce
 import uk.gov.hmrc.thirdpartyapplication.domain.models.ActorType._
 import uk.gov.hmrc.thirdpartyapplication.domain.models.State._
 import uk.gov.hmrc.thirdpartyapplication.services.ApplicationService
-import uk.gov.hmrc.thirdpartyapplication.domain.models.ImportantSubmissionData
 import uk.gov.hmrc.apiplatform.modules.submissions.services.SubmissionsService
 import uk.gov.hmrc.thirdpartyapplication.connector.EmailConnector
 import uk.gov.hmrc.http.HeaderCarrier
-
 import scala.concurrent.{ExecutionContext, Future}
-import cats.data.OptionT
 
 import javax.inject.Inject
 import java.time.{Clock, LocalDateTime}
 
-
-class ResponsibleIndividualVerificationService @Inject()(
-    responsibleIndividualVerificationRepository: ResponsibleIndividualVerificationRepository,
-    applicationRepository: ApplicationRepository,
-    stateHistoryRepository: StateHistoryRepository,
-    applicationService: ApplicationService,
-    submissionService: SubmissionsService,
-    emailConnector: EmailConnector,
-    declineApprovalsService: DeclineApprovalsService,
-    clock: Clock
- )(implicit ec: ExecutionContext) extends BaseService(stateHistoryRepository, clock) with ApplicationLogger {
-
-  def createNewVerification(applicationData: ApplicationData, submissionId: Submission.Id, submissionInstance: Int) = {
+class ResponsibleIndividualVerificationService @Inject()(responsibleIndividualVerificationRepository: ResponsibleIndividualVerificationRepository,
+                                                         applicationRepository: ApplicationRepository,
+                                                         stateHistoryRepository: StateHistoryRepository,
+                                                         applicationService: ApplicationService,
+                                                         submissionService: SubmissionsService,
+                                                         emailConnector: EmailConnector,
+                                                         declineApprovalsService: DeclineApprovalsService,
+                                                         clock: Clock)
+                                                        (implicit ec: ExecutionContext)
+  extends BaseService(stateHistoryRepository, clock) with ApplicationLogger {
```

Comment:
The formatting here, and in other files, seems to have gone crazy.
The previous version had the preferred formatting for constructor params.

Reply:
We've been using formatting using .scalafmt.conf in the project. We haven't changed the formatting on here.